legend-sdlc-generation-service/src/test/resources/generation/service/ModelToModelServiceWithMultipleParams.generated.java

```diff
@@ -0,0 +1,124 @@
+{
+    public ModelToModelServiceWithMultipleParams()
+    {
+        super("service::ModelToModelServiceWithMultipleParams", "org/finos/legend/sdlc/generation/service/entities/service/ModelToModelServiceWithMultipleParams.json", false);
+    }
+
+    public Result execute(java.lang.String i_s, long i_i, double i_f, java.math.BigDecimal i_dec, boolean i_b, java.time.LocalDate i_sd, java.time.ZonedDateTime i_dt, java.time.temporal.Temporal i_d, java.lang.Long i_oi, List<? extends java.lang.Long> i_li)
+    {
+        ExecutionBuilder builder = newExecutionBuilder(10);
+        if (i_s != null)
+        {
+            builder = builder.withParameter("i_s", i_s);
+        }
+        builder = builder.withParameter("i_i", i_i);
+        builder = builder.withParameter("i_f", i_f);
+        if (i_dec != null)
+        {
+            builder = builder.withParameter("i_dec", i_dec);
+        }
+        builder = builder.withParameter("i_b", i_b);
+        if (i_sd != null)
+        {
+            builder = builder.withParameter("i_sd", i_sd);
+        }
+        if (i_dt != null)
+        {
+            builder = builder.withParameter("i_dt", i_dt);
+        }
+        if (i_d != null)
+        {
+            builder = builder.withParameter("i_d", i_d);
+        }
+        if (i_oi != null)
+        {
+            builder = builder.withParameter("i_oi", i_oi);
+        }
+        if (i_li != null)
+        {
+            builder = builder.withParameter("i_li", i_li);
+        }
+        return builder.execute();
+    }
+
+    public Result execute(java.lang.String i_s, long i_i, double i_f, java.math.BigDecimal i_dec, boolean i_b, java.time.LocalDate i_sd, java.time.ZonedDateTime i_dt, java.time.temporal.Temporal i_d, java.lang.Long i_oi, List<? extends java.lang.Long> i_li, StreamProvider streamProvider)
+    {
+        ExecutionBuilder builder = newExecutionBuilder(10);
+        builder = builder.withStreamProvider(streamProvider);
+        if (i_s != null)
+        {
+            builder = builder.withParameter("i_s", i_s);
+        }
+        builder = builder.withParameter("i_i", i_i);
+        builder = builder.withParameter("i_f", i_f);
+        if (i_dec != null)
+        {
+            builder = builder.withParameter("i_dec", i_dec);
+        }
+        builder = builder.withParameter("i_b", i_b);
+        if (i_sd != null)
+        {
+            builder = builder.withParameter("i_sd", i_sd);
+        }
+        if (i_dt != null)
+        {
+            builder = builder.withParameter("i_dt", i_dt);
+        }
+        if (i_d != null)
+        {
+            builder = builder.withParameter("i_d", i_d);
+        }
+        if (i_oi != null)
+        {
+            builder = builder.withParameter("i_oi", i_oi);
+        }
+        if (i_li != null)
+        {
+            builder = builder.withParameter("i_li", i_li);
+        }
+        return builder.execute();
+    }
```

Comment:
Removing package on class parameters got very tricky, and reverted since not sure is worth the complexity.  I did removed packages from java.lang classes since are redundant, but left for other classes.